src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -112,7 +112,7 @@ const MoneyRequestAction = (props) => {
                 undefined,
                 CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS,
                 props.action.reportActionID,
-                props.chatReportID, // Needs to be changed to iouReportID
+                props.requestReportID,
```

Comment:
@luacmartins Can you please explain why was this needed? We think this is causing this bug https://github.com/Expensify/App/issues/19487

Reply:
We should not be building a thread when clicking the preview on a Split  action. We instead show a split details page that is being worked on [here](https://github.com/Expensify/App/pull/19390)

Reply:
it has to be for other cases, we should probably consider treating this a bit differntly for the split case, need to explore more there.